### ISSUE TITLE
Untrusted site block manual edit of url

### ISSUE THREAD

**Issue by Yankes** [hackademix/noscript#261]
I notice when I try edit url to have `https://localhost/` and current tab have page that is untrusted by NoScript,
then my action is blocked. When looking on F12 tab its say it is blocked by NoScript.

Could NoScript detect this is manual action and permit change of url in this case?
Or this is impossible for NoScript to detect how edit url?

**Comment by hackademix** [comment]
Seems a duplicate of #241 (fixed).
Feel free to reopen if it's not.
